test/unit/eigensolver/test_reduction_to_band.cpp

```diff
@@ -328,17 +344,35 @@ TYPED_TEST(ReductionToBandTestMC, CorrectnessLocal) {
   for (const auto& config : configs) {
     const auto& [size, block_size, band_size] = config;
 
-    testReductionToBandLocal<TypeParam, Device::CPU>(size, block_size, band_size);
+    testReductionToBandLocal<TypeParam, Backend::MC, Device::CPU>(size, block_size, band_size);
   }
 }
 
 TYPED_TEST(ReductionToBandTestMC, CorrectnessLocalSubBand) {
   for (const auto& config : configs_subband) {
     const auto& [size, block_size, band_size] = config;
 
-    testReductionToBandLocal<TypeParam, Device::CPU>(size, block_size, band_size);
+    testReductionToBandLocal<TypeParam, Backend::MC, Device::CPU>(size, block_size, band_size);
+  }
+}
+
+#ifdef DLAF_WITH_CUDA
+TYPED_TEST(ReductionToBandTestGPU, CorrectnessLocal) {
+  for (const auto& config : configs) {
+    const auto& [size, block_size, band_size] = config;
+
+    testReductionToBandLocal<TypeParam, Backend::GPU, Device::GPU>(size, block_size, band_size);
+  }
+}
+
+TYPED_TEST(ReductionToBandTestGPU, CorrectnessLocalSubBand) {
+  for (const auto& config : configs_subband) {
+    const auto& [size, block_size, band_size] = config;
+
+    testReductionToBandLocal<TypeParam, Backend::GPU, Device::GPU>(size, block_size, band_size);
   }
 }
+#endif
```

Comment:
I would keep this tests out until you have an implementation.

Reply:
Ok, so we can merge this independently from the implementation work.

In the meanwhile `master` will give a "not implemented" error in case this specific gpu function is going to be called.